oap-server/server-library/library-client/src/main/java/org/apache/skywalking/oap/server/library/client/elasticsearch/ElasticSearchClient.java

```diff
@@ -404,4 +434,65 @@ public String formatIndexName(String indexName) {
         }
         return indexName;
     }
+
+    private String[] formatIndexNames(String indexName, long startTimestamp, long endTimestamp, String traceId) throws IOException {
```

Comment:
Why include `traceId`? TraceId generation rule is only suitable in Java codebase. We shouldn't count on it at any time.

Reply:
The logic of determining the time according to the traceId can be removed, and different implementations of agents in other languages ​​were not considered at the time